alea/include/alps/alea/internal/joined.hpp

```diff
@@ -18,12 +18,49 @@ namespace alps { namespace alea {
 
     struct circular_var;
     struct elliptic_var;
+
 }}
 
```

Comment:
explanations?

Reply:
These are only forward declarations, the documentation is with the actual declaration.